### PR TITLE
CVSL-129: Convert cypress tests to typescript and write a test to click through the whole create a licence journey

### PR DESCRIPTION
**Pull request by smcveigh941**
* Converted existing cypress tests to typescript from JS
* Added pageId to page templates for reference in tests
* Disabled Axe test for additional conditions page - awaiting accessible design

### CODE REVIEW

File: integration_tests/gotenberg/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "noEmit": true,
```
Comment (timharrison-moj) [inline integration_tests/gotenberg/tsconfig.json:4]: Should this be ES6?

File: integration_tests/integration/createLicence.spec.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    const additionalConditionsQuestionPage = appointmentTimePage
+      .enterDate(moment().add(1, 'year'))
+      .enterTime(moment())
+      .clickContinue()
+
+    const additionalConditionsPage = additionalConditionsQuestionPage.selectYes()
+
+    const bespokeConditionsQuestionPage = additionalConditionsPage.clickContinue()
+
```
Comment (timharrison-moj) [inline integration_tests/integration/createLicence.spec.ts:41]: Is this right? bespokeConditionsQuestionPage = additionalConditionsPage.clickContinue??

Comment (timharrison-moj) [reply integration_tests/integration/createLicence.spec.ts:41]: It might well be..

File: integration_tests/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "noEmit": true,
```
Comment (timharrison-moj) [inline integration_tests/tsconfig.json:4]: ES6?

File: server/views/pages/create/initialMeetingPerson.njk [hunk 4]
```diff
@@ -18,14 +19,19 @@
                 {{ govukInput({
                     id: "contactName",
                     name: "contactName",
+                    label: {
+                        text: "contact name",
+                        classes: "govuk-visually-hidden"
+                    },
```
Comment (timharrison-moj) [inline server/views/pages/create/initialMeetingPerson.njk:25]: Is there a reason for hiding the label here?

**Review by timharrison-moj — APPROVED** [review]
LGTM. A couple of minor questions only. Nice work.

File: integration_tests/integration/createLicence.spec.ts (hunk 2 above)
Comment (smcveigh941) [reply integration_tests/integration/createLicence.spec.ts:41]: Label is hidden for accessibility - Axe test failed. We dont want to see it but screen reader will see it

File: server/views/pages/create/initialMeetingPerson.njk (hunk 4 above)
Comment (smcveigh941) [reply server/views/pages/create/initialMeetingPerson.njk:25]: Label is hidden for accessibility - Axe test failed. We dont want to see it but screen reader will see it

**Pull request merged**